regexes/device/mobiles.yml

```diff
@@ -6122,8 +6122,6 @@ OnePlus:
       model: '5'
     - regex: 'A600[03]|OnePlus6'
       model: '6'
-    - regex: 'A6013'
-      model: '6T'
```

Comment:
Feel free to readd the detection for OnePlus 6T. To get the test pass you need to write `brand: "ON"`, A `ON` without `"` will be converted to true by the YAML parser...